specification/sql/resource-manager/readme.md

```diff
@@ -131,6 +131,7 @@ input-file:
  - ./Microsoft.Sql/2014-04-01/importExport.json
  - ./Microsoft.Sql/2014-04-01/metrics.json
  - ./Microsoft.Sql/2014-04-01/operations.json
+ - ./Microsoft.Sql/2014-04-01/queries.json

```

Comment:
just wondering should 'schema-2014-04' be the new default tag?  - see line 31 in this file.